amethyst_controls/src/bundles.rs

```diff
@@ -27,33 +29,20 @@ use super::*;
 /// * `FreeRotationSystem`
 /// * `MouseFocusUpdateSystem`
 /// * `CursorHideSystem`
-#[derive(Debug)]
+#[derive(new, Debug)]
 pub struct FlyControlBundle<T: BindingTypes> {
+    #[new(value = "1.0")]
     sensitivity_x: f32,
+    #[new(value = "1.0")]
     sensitivity_y: f32,
+    #[new(value = "one()")]
```

Comment:
This should be 1.0 too actually. It seems we forgot to change it following the removal of the `Float` type.